src/styles/components/breadcrumbs/styles.less

```diff
@@ -1,144 +1,312 @@
 /* -----------------------------------------------------------------------------
 --------------------------------------------------------------------------------
-
-Breadcrumbs
-
+--------------------------------------------------------------------------------
+breadcrumbs/styles.less
+--------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 ----------------------------------------------------------------------------- */
 
-.breadcrumbs {
+& when (@breadcrumbs-enabled) {
 
-  &.inverse {
+  .breadcrumbs {
 
-    a {
-      color: inherit;
+    &.inverse {
 
-      &:hover {
-        color: color-lighten(@neutral, 20);
-        text-decoration: none;
-      }
+      a {
+        color: inherit;
+
+        &:hover {
+          color: color-lighten(@neutral, 20);
+          text-decoration: none;
+        }
+
+        &:active {
+          color: color-lighten(@neutral, 30);
+        }
 
-      &:active {
-        color: color-lighten(@neutral, 30);
       }
+
     }
-  }
 
-  .icon {
-    opacity: 0.3;
-    margin-right: @base-spacing-unit * 0.5;
-    margin-left: @base-spacing-unit * 0.5;
-  }
+    .icon {
+      opacity: 0.3;
+      margin-right: @base-spacing-unit * 0.5;
+      margin-left: @base-spacing-unit * 0.5;
+    }
 
-}
+  }
 
-.breadcrumb {
-  display: flex;
-  flex: 0 0;
-  font-size: @base-spacing-unit * .65;
-  line-height: @base-spacing-unit;
-  list-style: none !important;
-  padding-top: 0; // this + container-pod-short-top = 26px;
-  padding-bottom: @base-spacing-unit * 2/3;
-  margin: 0;
-
-  li {
+  .breadcrumb {
     display: flex;
+    flex: 0 0;
+    font-size: @body-font-size;
```

Comment:
We should do a `null` check on these either manually or using the `property-variant()`.  Looks like we do it further down in the media queries, just not here.